src/stdlib_math.fypp

```diff
@@ -409,17 +416,18 @@ contains
         ${t1}$, intent(in) :: z
         real(${k1}$) :: result
 
-        result = merge(0.0_${k1}$, atan2(z%im, z%re), z == (0.0_${k1}$, 0.0_${k1}$)) &
-                 *180.0_${k1}$/PI_${k1}$
+        result = merge(0.0_${k1}$, atan2(z%im, z%re)*180.0_${k1}$/PI_${k1}$, &
+                 z == (0.0_${k1}$, 0.0_${k1}$))
 
```

Comment:
`argd`: Putting the multiplication into the `merge` can reduce the unnecessary computation when `z=(0,0)`